build.sbt

```diff
@@ -222,3 +222,19 @@ lazy val apiModel = (project in file("api-model"))
     libraryDependencies ++= commonDeps ++ httpDeps
   ).dependsOn(modelJVM % "compile->compile;test->test")
   .dependsOn(persistence % "compile->compile")
+
+lazy val publication = (project in file("publication"))
```

Comment:
Good question. It's one that we need @jmarin to answer. He wrote `build.sbt` this way, so I'd say let's leave this cod as-is for now, and make a ticket to remind us to confirm.